app/utils/images.ts

```diff
@@ -13,8 +13,10 @@ import {
     VIEWPORT_IMAGE_REPLY_OFFSET,
 } from '@constants/image';
 import {isImage} from './file';
+import {PostImage} from '@mm-redux/types/posts';
+import {FileInfo} from '@mm-redux/types/files';
 
-export const calculateDimensions = (height, width, viewPortWidth = 0, viewPortHeight = 0) => {
+export const calculateDimensions = (height: number, width: number, viewPortWidth = 0, viewPortHeight = 0) => {
```

Comment:
After looking in the code, I saw that in file_attachments.js we are using it as:
`calculateDimensions(file?.height, file?.width, wrapperWidth, viewPortHeight);`
Therefore, it may be undefined. I changed it.